pkg/services/sqlstore/migrations/accesscontrol/dashboard_permissions.go

```diff
@@ -408,6 +408,15 @@ func AddManagedFolderAlertActionsRepeatMigration(mg *migrator.Migrator) {
 	mg.AddMigration(managedFolderAlertActionsRepeatMigratorID, &managedFolderAlertActionsRepeatMigrator{})
 }
 
+const managedFolderAlertActionsRepeatMigratorFixedID = "managed folder permissions alert actions repeated fixed migration"
+
+/*
+AddManagedFolderAlertActionsRepeatFixedMigration is a fixed version of AddManagedFolderAlertActionsRepeatMigration.
+*/
+func AddManagedFolderAlertActionsRepeatFixedMigration(mg *migrator.Migrator) {
```

Comment:
I think this function should be called from somewhere here: https://github.com/grafana/grafana/blob/main/pkg/services/sqlstore/migrations/migrations.go#L106

Reply:
sorry had that checked in previously, missed to check in after some changes

Reply:
https://github.com/grafana/grafana/pull/58054/files#diff-0fd985df1b8aaadcea6def96014b31f7eee27c4ea19761733754bffcf7d06f6aR106